multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -691,6 +699,27 @@ def _setup_distillation(
         else:
             raise ValueError(f"Unknown soft_label_loss_type: {self._config.distiller.soft_label_loss_type}")
 
+        if self._config.distiller.output_feature_loss_type == "cosine":
```

Comment:
We may use the following to avoid backward incompatibility with 0.5.

```python3
output_feature_loss_type = OmegaConf.select(self._config, "distiller.output_feature_loss_type", default="mse")

if output_feature_loss_type == "cosine":
...

```

Reply:
Done